assets/app/styles/_mixins.less

```diff
@@ -36,4 +36,29 @@
  -ms-word-break: break-all; // ms
      word-break: break-all; // firefox
      word-break: break-word; // webkit
-  }
\ No newline at end of file
+  }
+
+// Missing from our current version of bootstrap
+.transition-timing-function(@timing-function) {
+  -webkit-transition-timing-function: @timing-function;
+          transition-timing-function: @timing-function;
+}
+
+// Linear-gradient stripes 
+.striped(@color: rgba(255,255,255,.15); @angle: 45deg) {
+  background-image: -webkit-linear-gradient(@angle, @color 25%, transparent 25%, transparent 50%, @color 50%, @color 75%, transparent 75%, transparent);
+  background-image: linear-gradient(@angle, @color 25%, transparent 25%, transparent 50%, @color 50%, @color 75%, transparent 75%, transparent);
+}
+
+// Our version of less doesn't support comma separated values for transition properties. 
+// A solution is http://alittlecode.com/handling-multiple-css3-transitions-with-a-less-mixin/
+.transition-property (@value1,@value2:X,...)
+{
+    @value: ~`"@{arguments}".replace(/[\[\]]|\,\sX/g, '')`;
```

Comment:
@sg00dwin This makes me nervous. Do escaped strings work? For instance,

```
.transition-property(~"border-color, background-image, background-color");
```

and

```
.transition-property (@property) {
 -webkit-transition-property: @property;
// etc
}
```